Cargo.toml

```diff
@@ -26,3 +26,5 @@ oci_spec = { version = "0.1.0", path = "./oci_spec" }
 [dev-dependencies]
 oci_spec = { version = "0.1.0", path = "./oci_spec", features = ["proptests"] }
 quickcheck = "1"
+serial_test = "0.5.1"
+tempfile = "3.2.0"
```

Comment:
How about moving the one used in the cgroups test to a more generic location and using it?
https://github.com/containers/youki/blob/1afde706fbfb94715f554f7fe7a31bb0eb86206c/src/cgroups/test.rs#L13

Reply:
Moved the TempDir implmentation onto utils